ukis_pysat/raster.py

```diff
@@ -360,13 +361,17 @@ def to_dask_array(self, chunk_size=(1, 6000, 6000)):
         self.da_arr = da.from_array(self.__arr, chunks=chunk_size)
         return self.da_arr
 
-    def write_to_file(self, path_to_file, dtype=rasterio.uint16, driver="GTiff"):
+    def write_to_file(self, path_to_file, dtype, driver="GTiff", compress=None):
         """
         Write a dataset to file.
         :param path_to_file: str, path to new file
-        :param dtype: datatype, optional (default: rasterio.uint16)
+        :param dtype: datatype, like rasterio.uint16, 'float32' or 'min' to use the minimum type to represent values
         :param driver: str, optional (default: 'GTiff')
+        :param compress: compression, e.g. 'lzw' (default: None)

```

Comment:
we could add "tiled" and "nodata" as further optional parameters here. could be useful for certain applications and does not do any harm here.

Reply:
If we offer everything the user could also just use rasterio for this purpose. Let's keep it simple.

Reply:
ok agree with that.

Reply:
Please check one more time the tests. Just reran them and seems to fail when checking compression result.

Reply:
Whatever caused this behaviour, the new test is better.